Gruntfile.coffee

```diff
@@ -1,5 +1,40 @@
 module.exports = (grunt) ->
   grunt.initConfig
+    
+    # Delete files that will be used by compile-handlebars because it appends to the end of a file rather than re-writing it.
+    clean: [ 'src/sass/_*-map.scss', 'index.html']
```

Comment:
Might as well remove `_*-map.scss` from git if they're only going to be used for generating CSS.  That way we aren't tracking as much churn when the source JSON changes.

Reply:
Might they be needed for anyone using SCSS rather than the compiled CSS?

Reply:
That's true, though I'm not sure if anyone's actually using that.  Safe to leave in for now.